CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ set( CMAKE_ALLOW_LOOSE_LOOP_CONSTRUCTS TRUE )
 
 # Define here the needed parameters
 set (OPENRAVE_VERSION_MAJOR 0)
-set (OPENRAVE_VERSION_MINOR 129)
+set (OPENRAVE_VERSION_MINOR 130)

```

Comment:
please increment patch version only

Reply:
@yoshikikanemoto changed to increment patch version